src/main/java/com/mgmtp/radio/domain/station/Song.java

```diff
@@ -0,0 +1,34 @@
+package com.mgmtp.radio.domain.station;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@NoArgsConstructor
+@Data
+@Document(collection = "song")
+public class Song {
+	@Id
+	private String id;
```

Comment:
please add songId (eg: youtube song id), source (= youtube) so that we can extend in future